tests/providers/amazon/aws/operators/test_datasync.py

```diff
@@ -69,13 +68,9 @@
 
 @mock_datasync
 @mock.patch.object(DataSyncHook, "get_conn")
-class DataSyncTestCaseBase(unittest.TestCase):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.datasync = None
-
+class DataSyncTestCaseBase:
     # Runs once for each test
-    def setUp(self):
+    def setup(self):
```

Comment:
I've found that `pytest` actually run methods named  **setup**,  **teardown**. I don't know might be it some kind of undocumented feature which could remove in future releases. So better keep general notation

```suggestion
    def setup_method(self, method):
```